frappe/public/js/frappe/ui/toolbar/awesome_bar.js

```diff
@@ -29,16 +29,18 @@ frappe.search.AwesomeBar = class AwesomeBar {
 				};
 			},
 			item: function (item, term) {
-				var d = this.get_item(item.value);
-				var name = __(d.label || d.value);
-				var html = "<span>" + name + "</span>";
+				const d = this.get_item(item.value);
+				const target = d.route ? frappe.router.make_url(d.route) : "#";
```

Comment:
frappe.router.make_url only handles route passed as an Array e.g ["Workspaces", "private", "Home"] and not string e.g. "List/Workspaces/List"

Reply:
So in the case of strings, should we just parse them first? I think I've seen a util for this. Or put them directly into the href?

Reply:
> So in the case of strings, should we just parse them first? I think I've seen a util for this. Or put them directly into the href?

It is handled in `frappe.set_route` method but I don't think there is a separate util just for this

Reply:
This works: 
```js
frappe.router.get_route_from_arguments(["List/Workspaces/List"])
// ['List', 'Workspaces', 'List']
```